### PR TITLE
Maticx refactor

### PR DESCRIPTION
**Pull request by manoj9april**
- removed permissions from `run-scenarios.yaml`
- minor refactoring

### CODE REVIEW

File: deployments/polygon/usdc/migrations/1689168483_add_maticx_collateral.ts [hunk 1]
```diff
@@ -98,6 +96,10 @@ export default migration("1689168483_add_maticx_collateral", {
     trace(`Created proposal ${proposalId}.`);
   },
 
+  async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
+    return true;
+  },
+
```
Comment (kevincheng96) [inline deployments/polygon/usdc/migrations/1689168483_add_maticx_collateral.ts:102]: This shouldn't be true unless the proposal was actually created.

**Comment by kevincheng96** [comment]
Can you remove the unrelated commits?

File: deployments/polygon/usdc/migrations/1689168483_add_maticx_collateral.ts (hunk 1 above)
Comment (manoj9april) [reply deployments/polygon/usdc/migrations/1689168483_add_maticx_collateral.ts:102]: cool will remove

**Review by kevincheng96 — APPROVED** [review]
LGTM

**Pull request merged**